source/includes/_tipser-react-elements.md

```diff
@@ -1,79 +1,75 @@
 # Tipser React Elements
-Tipser Elements is a React components library from Tipser enabling you to add shoppable content to your site with minimal effort.  You can add simple components like products with buy buttons, collections of products and articles, but also more complex grids or whole pages.
+Tipser React Elements is a component library from Tipser enabling you to add shoppable content to your React APP with minimal effort. You can add simple components like products with buy buttons, collections of products and articles, but also more complex grids or whole pages. 
 
-The package allows you to use Tipser's product repository and create inline shops embedded into your page as well as the entire websites.
+<aside class="notice">
+Not using React? Or wanted to learn what Tipser Elements are generally? Go check <a href="#tipser-elements">Tipser Elements</a> docs. This section covers only <strong>React API</strong> for Elements. 
```

Comment:
`Or wanted to learn what Tipser Elements are generally` - not sure if this sentence is needed at all, if so we can replace it with someting like: `...or do you want to get a general overview of Tipser Elements?`

Reply:
Done.